nomad/structs/structs.go

```diff
@@ -10632,6 +10637,27 @@ type Evaluation struct {
 	ModifyTime int64
 }
 
+type EvaluationStub struct {
+	ID                string
+	Namespace         string
+	Priority          int
+	Type              string
+	TriggeredBy       string
+	JobID             string
+	NodeID            string
+	DeploymentID      string
+	Status            string
+	StatusDescription string
+	WaitUntil         time.Time
+	NextEval          string
+	PreviousEval      string
+	BlockedEval       string
+	CreateIndex       uint64
+	ModifyIndex       uint64
+	CreateTime        int64
+	ModifyTime        int64
+}
```

Comment:
This list is fairly arbitrary and will be mostly dictated by the UI. `Status` and `StatusDescription` are probably the most important right now, but since this is only returned in an opt-in request I added most of the fields.

Using an `Evalution` directly in `RelatedEvals` caused a recursive issue while decoding the struct.